common/src/constants/grade-boundaries.ts

```diff
@@ -221,3 +221,25 @@ export const ONGEKI_GBOUNDARIES = MakeGradeBoundaries<GetEnumValue<"ongeki:Singl
 	SSS: 1_000_000,
 	"SSS+": 1_007_500,
 });
+
+type DDRLikes = GPTStrings["ddr"];
```

Comment:
The only reason we have `IIDXLikes` and `SDVXLikes` is because BMS and USC happen to share those scoring systems.

DDR just -is- DDR. You can remove this type declaration and just write `GPTStrings["ddr"]` out.